Database/Corrections/ContentPhases/Anniversary.lua

```diff
@@ -775,9 +775,14 @@ local questsToBlacklistByPhase = {
 ---@param contentPhase number
 ---@return table<QuestId, boolean>
 function ContentPhases.BlacklistAnniversaryQuestsByPhase(questsToBlacklist, contentPhase)
+    if not questsToBlacklistByPhase or not contentPhase then
+        return questsToBlacklist
+    end
     for phase = contentPhase + 1, #questsToBlacklistByPhase do
-        for questId in pairs(questsToBlacklistByPhase[phase]) do
-            questsToBlacklist[questId] = true
+        if questsToBlacklistByPhase[phase] then
+            for questId in pairs(questsToBlacklistByPhase[phase]) do
+                questsToBlacklist[questId] = true
+            end
```

Comment:
With a proper solution/work around for `Questie.IsTBC` this isn't needed.

In general we plan to do some renaming so that the current "Anniversary" will become "Era" and it can be better distinguished to "TBC" (there won't be "Anniversary Era" one TBC goes live, will it?). I don't think there will be "Anniversary TBC" naming wise as there is no competitive TBC flavor.